src/openforms/analytics_tools/tests/test_migrations.py

```diff
@@ -0,0 +1,71 @@
+from openforms.utils.tests.test_migrations import TestMigrations
+
+
+class UpdateSourceIDsGovMetricTests(TestMigrations):
+    app = "analytics_tools"
+    migrate_from = "0006_auto_20240112_1046"
+    migrate_to = (
+        "0009_remove_analyticstoolsconfiguration_govmetric_secure_guid_and_more"
+    )
+
+    def setUpBeforeMigration(self, apps):
+        AnalyticsToolsConfiguration = apps.get_model(
+            "analytics_tools", "AnalyticsToolsConfiguration"
+        )
+        AnalyticsToolsConfiguration.objects.create(
+            govmetric_source_id_en="1234",
+            govmetric_source_id_nl="7890",
+            govmetric_secure_guid_en="4321",
+            govmetric_secure_guid_nl="0987",
+        )
+
+    def test_builder_enabled(self):
```

Comment:
Where do we exaclty refer to with the `builder` word?

Reply:
Caught copy-pasting :see_no_evil: